source/faucet/main.d

```diff
@@ -117,7 +117,7 @@ private struct State
 
                 foreach (ref b; blocks)
                     foreach (ref tx; b.txs)
-                        if (tx.type == TxType.Payment)
+                        if (tx.isPayment)
```

Comment:
You could just introduce those functions in a standalone PR in Agora, then update Faucet, and break the cycle.
Alternatively, we could just add all UTXOs (even frozen ones), and filter when we draw from it.

Reply:
Good idea, think I will go for the first option.